.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: trusty
 language: node_js
 
 node_js:
-  - 6
+  - 8.11
 
```

Comment:
Remove matrix, include, and services and shift the content to left align and remove ```-``` from left side of pameters(before_script and script).